g9frontend/src/components/Register/Register.css

```diff
@@ -0,0 +1,64 @@
+
+.register-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+
+.top-bar {
+    width: 100%;
+    height: 50px;
+    background-color: #2f668d;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 20px;
+    color: white;
+}
+
+
+.register-form {
+    margin-top: 50px;
+    text-align: center;
+}
+
+.register-form h2 {
+    margin-bottom: 20px;
+}
+
+.input-section {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+}
+
+.input-section input {
+    width: 300px;
+    height: 30px;
+    margin-bottom: 10px;
+    padding: 5px;
+    border-radius: 5px;
+    border: 1px solid #ccc;
+    box-sizing: border-box;
+    background-color: #d9d6d6;
+    font-size: 16px;
+    display : block;
+}
+.input-section input::placeholder {
+    color: white;
+}
+
+button {
+    width: 120px;
+    height: 40px;
+    background-color: #2f668d;
```

Comment:
For some reason, this line modified the colour of the Toggle Buttons inside the MainPage as well. Please modify this so that only that specific button gets this colour. You can do something similiar to what you did on line `46` inside `Login.css`.